core/layouts/geary/layout--geary.tpl.php

```diff
@@ -31,82 +31,92 @@
 
   <?php if ($content['header']): ?>
     <header class="l-header" role="banner" aria-label="<?php print t('Site header'); ?>">
-      <div class="container container-fluid">
-        <div class="row">
-          <?php print $content['header']; ?>
-        </div><!-- /.row -->
+      <div class="l-header-inner container container-fluid">
+        <?php print $content['header']; ?>
       </div><!-- /.container -->
     </header>
   <?php endif; ?>
 
-  <div class="l-wrapper">
-    <div class="l-wrapper-inner container container-fluid">
+  <section class="l-wrapper">
```

Comment:
I think every layout's content area wrapper should be a section tag. We can't say for sure that it should be an article tag, and this is a good semantic step up from a div.